lib/diagnostic.js

```diff
@@ -22,13 +25,22 @@ const uniqueWords = function (/* */) {
 // a second argument.
 
 const wordCount = function (/* */) {
-  /* your response here */
+  for (let i = 0; i < normalizedWords.length; i++) {
+      let word = normalizedWords[i];
+      wordCount[word] = addOne(wordCount[word]);
+      }
 }
 
 // 4. Write a function that takes a string and returns a dictionary with unique
 // words as keys and a count of each word as the value for that key
 const wordFrequencies = function (/* */) {
-  /* your response here */
+  for (let i = 0; i < wordFrequencies.length; i++) {
+    if (wordFrequencies[normalizedWords[i]] === undefined) {
```

Comment:
Make sure you are actually creating an object somewhere called `wordFrequencies`, otherwise this will not work. Also, it's probably a good idea not to give objects the same name as the functions that contain them.